source/sessions/tests/driver-sessions-argument-for-correct-client.yml

```diff
@@ -0,0 +1,313 @@
+description: driver-sessions-argument-for-correct-client
+
+schemaVersion: "1.0"
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - client:
+      id: &client1 client1
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name session-tests
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name test
+  - session:
+      id: &session1 session1
+      client: *client1
+
+tests:
+  - description: Session argument is for the right client (insertOne)
```

Comment:
It's expensive to setup and teardown each of these tests. Could we combine all of these tests into one to avoid that problem?